tests/flows/conftest.py

```
@@ -203,3 +203,41 @@ def mock_notify_agreement_unhandled_exception_in_teams(mocker):
+
+
+@pytest.fixture
+def mock_notify_missing_discount_levels(mocker):
+    return mocker.patch(
+        "adobe_vipm.flows.utils.subscription.notify_discount_level_error", spec=True
+    )
+
+
+@pytest.fixture
+def mock_get_template_data_by_adobe_subscription(mocker):
+    mock = mocker.MagicMock(
```

Flake8 (7.3.0)

[error] local variable 'mock' is assigned to but never used

(F841)